e2e_test/udf/test.py

```diff
@@ -0,0 +1,29 @@
+import sys
+sys.path.append('src/udf/python')  # noqa
```

Comment:
What is `noqa` 🥵

Reply:
> Adding # noqa to a line indicates that the linter (a program that automatically checks code quality) should not check this line. Any warnings that code may have generated will be ignored.

WOW 😲

Reply:
If I don't add this hint, the formatter will reorder this line below all imports, which messes things up. 😇